ricecooker/utils/youtube.py

```diff
@@ -337,12 +338,16 @@ def get_language_with_alpha2_fallback(language_code):
     using the `getlang_by_alpha2` helper method.
     Returns either a le-utils Language object or None if both lookups fail.
     """
-    # 1. try to lookup `language` using internal representation
-    language_obj = languages.getlang(language_code)
-    # if language_obj not None, we know `language` is a valid language_id in the internal repr.
-    if language_obj is None:
-        # 2. try to match by two-letter ISO code
-        language_obj = languages.getlang_by_alpha2(language_code)
+    try:
```

Comment:
The docstring here still refers the `getlang_by_alpha2` function - maybe should be updated to match the new implementation?